website/pages/docs/plugins/destinations/clickhouse/configuration.mdx

```diff
@@ -0,0 +1,67 @@
+# ClickHouse destination plugin configuration reference
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+import { Callout } from 'nextra-theme-docs'
+
+<Badge text={"Latest: " + getLatestVersion("destination", "clickhouse")}/>
+
+<Callout type="info">
+Make sure you use environment variable expansion in production instead of committing the credentials to the configuration file directly.
+</Callout>
```

Comment:
This callout should probably be lower down (below the example), it doesn't need to be the first thing on the page